docs/src/pages/guides/interoperability/interoperability.md

```diff
@@ -734,3 +737,103 @@ export default function SliderThumbOverrides() {
   return <Slider className="p-4" classes={{ active: 'shadow-none' }} />;
 }
 ```
+
+## JSS
```

Comment:
I don't mind having section about `tss-react`, but having it under JSS header is misleading, as it is not using JSS, but emotion. I would rather add the usage of `tss-react` in a subsection under emotion maybe (I am not sure about this). cc @mui-org/core

Reply:
Hi @mnajdova,  
  
I am afraid that mentioning TSS as a subsection of `@emotion` could be confusing for some peoples.
 The fact that TSS uses emotion under the hood is merely a technicality.  The API of `tss-react` is much closer to `react-jss` than it is from the API of emotion.     

On the other hand, I think that peoples that would like to use a JSS like API would be interested to know that there is an option for them.  

What do you think of my update? I replaced the title by:  ~~JSS~~ TSS.  

Best regards,  

PS: `tss-react` now has a proper documentation website: https://docs.tss-react.dev